bindings/gumjs/runtime/cmodule/ptrauth.h

```diff
@@ -0,0 +1,45 @@
+#include <stdint.h>
+
+enum _ptrauth_key
+{
+  ptrauth_key_asia = 0,
+  ptrauth_key_asib = 1,
+  ptrauth_key_asda = 2,
+  ptrauth_key_asdb = 3,
+  ptrauth_key_process_independent_code = ptrauth_key_asia,
+  ptrauth_key_process_dependent_code = ptrauth_key_asib,
+  ptrauth_key_process_independent_data = ptrauth_key_asda,
+  ptrauth_key_process_dependent_data = ptrauth_key_asdb,
+  ptrauth_key_function_pointer = ptrauth_key_process_independent_code,
+  ptrauth_key_return_address = ptrauth_key_process_dependent_code,
+  ptrauth_key_frame_pointer = ptrauth_key_process_dependent_data,
+  ptrauth_key_block_function = ptrauth_key_asia,
+  ptrauth_key_cxx_vtable_pointer = ptrauth_key_asda,
+};
+
+typedef enum _ptrauth_key ptrauth_key;
+typedef uintptr_t ptrauth_extra_data_t;
+typedef uintptr_t ptrauth_generic_signature_t;
+
+#ifdef HAVE_PTRAUTH
+
+# define ptrauth_strip(__value, __key) \
```

Comment:
```suggestion
# define ptrauth_strip(value, key) \
```

By dropping the underscore prefixes here and below it should be easier to format the code below to avoid exceeding 80 cols.